packages/jest-types/src/Config.ts

```diff
@@ -278,13 +278,16 @@ type DisplayNameColor =
   | 'bgCyanBright'
   | 'bgWhiteBright';
 
+export type CoverageThresholdValue = {
+  branches?: number | null;
+  functions?: number | null;
+  lines?: number | null;
+  statements?: number | null;
```

Comment:
when these can be null?

Reply:
https://github.com/yordis/jest/blob/2440f830f1a816088943a39162dd9595d8803ee6/packages/jest-reporters/src/coverage_reporter.ts#L229

Unless I am misunderstanding something because I thought the same.

Reply:
This check is for null and undefined. Let's switch it to only check for undefined and simplify this type, we don't document using null anywhere AFAICT. cc @SimenB

Reply:
Perfect! Let me fix the typespec and the statement.